packages/web-react/docs/stories/examples/FooterContent.stories.tsx

```diff
@@ -0,0 +1,370 @@
+                  <li>
+                    <Link href="https://www.example.com">Link</Link>
+                  </li>
+                  <li>
+                    <Link href="https://www.example.com">Link</Link>
+                  </li>
+                  <li>
+                    <Link href="https://www.example.com">Link</Link>
+                  </li>
+                  <li>
+                    <Link href="https://www.example.com">Link</Link>
+                  </li>
+                  <li>
+                    <Link href="https://www.example.com">Link</Link>
+                  </li>
+                  <li>
+                    <Link href="https://www.example.com">Link</Link>
+                  </li>
+                </Stack>
+              </nav>
+            )}
+
+            {/* Navigation links 4 */}
+            {numberOfLinkColumns === 4 && (
+              <nav aria-labelledby="footer-navigation-section-4">
+                <Text
+                  id="footer-navigation-section-4"
+                  elementType="h3"
+                  size="large"
+                  emphasis="bold"
+                  marginBottom="space-600"
+                >
+                  {columnHeading('Section headline')}
+                </Text>
+                <Stack elementType="ul" spacing="space-500" hasSpacing>
+                  <li>
+                    <Link href="https://www.example.com">Link</Link>
+                  </li>
+                  <li>
+                    <Link href="https://www.example.com">Link</Link>
+                  </li>
+                  <li>
+                    <Link href="https://www.example.com">Link</Link>
+                  </li>
+                </Stack>
+              </nav>
+            )}
+          </Grid>
+        )}
+
+        {/* Divider */}
+        {showDividers && numberOfLinkColumns > 0 && <Divider marginY="space-900" />}
+
+        {/* Grid with product logo, social media links and language switch */}
+        <Grid
+          cols={{ mobile: 1, desktop: logoRowColumns() }}
+          alignmentX={{ mobile: 'center', desktop: 'stretch' }}
+          alignmentY="center"
+          spacing="space-900"
+          marginBottom={showDividers || (!showDividers && !showSecondaryLinks) ? undefined : 'space-900'}
+        >
+          {/* Product logo */}
+          <div className={showSocialMediaButtons || showLanguageSwitch ? 'text-desktop-left' : 'text-center'}>
+            <Link href="https://www.example.com">
+              <UNSTABLE_ProductLogo>{defaultSvgLogo}</UNSTABLE_ProductLogo>
+            </Link>
+          </div>
+
+          {/* Flex with social media links */}
+          {showSocialMediaButtons && (
+            <Flex elementType="ul" alignmentX={showLanguageSwitch ? 'center' : 'right'} alignmentY="center">
+              <li>
+                <ButtonLink size="medium" color="secondary" isSquare>
+                  <span className="accessibility-hidden">Facebook</span>
```

Comment:
we have a VisuallyHidden component
```suggestion
                  <VisuallyHidden>Facebook</VisuallyHidden>
```